cmd/installer/cli/install.go

```diff
@@ -171,6 +172,9 @@ func preRunInstall(cmd *cobra.Command, flags *InstallCmdFlags) error {
 		return fmt.Errorf("install command must be run as root")
 	}
 
+	// set the umask to 022 so that we can create files/directories with 755 permissions
+	_ = syscall.Umask(0o022)
```

Comment:
can you at the very least logrus.Debug this error?

Reply:
it doesn't return an error, it returns the previous umask value